src/System.IO.Compression/tests/ZipArchive/zip_ReadTests.cs

```diff
@@ -9,6 +9,22 @@ namespace System.IO.Compression.Tests
 {
     public class zip_ReadTests : ZipFileTestBase
     {
+        //[Fact]
+        //public static void ReadHugeFileWithoutEOCD()
+        //{
+        //    Diagnostics.Stopwatch sw = new Diagnostics.Stopwatch();
```

Comment:
We cannot test with stopwatch during CI.  No matter how much we relax this it would still be flaky.  Instead have a test stream that reports how much data actually was read out of it, then assert that less than some amount was read.

Reply:
Thanks for letting me know, I understand, I wasn't planning on leaving this. I was just testing the elapsed time to compare with the original reports (tens of seconds, or even minutes).